src/vm/functions/mod.rs

```diff
@@ -42,8 +42,8 @@ use vm::{eval, Environment, LocalContext};
 
 use address::AddressHashMode;
 use chainstate::stacks::{StacksAddress, C32_ADDRESS_VERSION_TESTNET_SINGLESIG};
-pub use vm::functions::assets::{get_stx_balance_snapshot, stx_transfer_consolidated};
-pub use vm::functions::special::handle_contract_call_special_cases;
+pub use vm::functions::assets::stx_transfer_consolidated;
+pub use vm::functions::special::{handle_contract_call_special_cases, handle_poison_microblock};
```

Comment:
Not necessarily the right place for this feedback / question.
From a smart contract point of view, I think it would be interesting to expose the microblock height / sequence number in `get-block-info?`. Thought?

Reply:
I'm open to it, but it would be a separate PR.